posthog/tasks/email.py

```diff
@@ -117,6 +118,12 @@ def send_email_verification(user_id: int, token: str) -> None:
             "site_url": settings.SITE_URL,
         },
     )
+    posthoganalytics.capture(
+        user.distinct_id,
+        "verification email sent",
+        properties={"email address": user.email},
```

Comment:
We tend to snake_case property names, so best to stick with that convention.

However, I don't see how this is necessary, since the email will be associated with the user anyway? I think we can do without.

Reply:
You're right, it should be pulled in with the distinct_id. Removing